cli/cmd/brokers.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(brokersCmd)
```

Comment:
Do you see us having more than one top-level cmd (I don't consider "version" a real cmd)? If not do we need to require the "brokers" word on the cmd line?